technical-reports/format/groups.md

```diff
@@ -0,0 +1,180 @@
+# Groups
+
+A file may contain many tokens and they may be nested arbitrarily in groups like so:
+
+```
+{
+  "token uno": {
+    "value": "token value 1"
+  },
+  "token group": {
+    "token dos": {
+      "value": "token value 2"
+    },
+    "nested token group": {
+      "token tres": {
+        "value": "token value 3"
+      },
+      "Token cuatro": {
+        "value": "token value 4"
+      }
+    }
+  }
+}
+```
```

Comment:
Can you wrap this block into an example aside element? (see my other suggestion above)